shared/common-adapters/icon.desktop.js

```diff
@@ -92,8 +92,7 @@ class Icon extends Component<Props, void> {
           fontWeight: 'normal',
           fontVariant: 'normal',
           textTransform: 'none',
-          lineHeight: 'initial',
-          verticalAlign: 'bottom',
+          lineHeight: 1,
```

Comment:
This was always the correct CSS to apply to the icon-font elements. `line-height: number` is a scalar of the element's `font-size`. So for a small icon we render with `font-size: 8px` and `line-height: 1` will make that icon's line height `8px` as well.

With the descent calculations in `font.js` this should always position the icon-font within 1px of the baseline of the text it is rendered next to.

Reply:
https://icomoon.io/#docs/font-metrics